src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -694,14 +699,13 @@ private void executeCreateReferenceDatabases(long personID)
 	{
 		ReferenceDatabaseCreator creator = new ReferenceDatabaseCreator(this.dbAdapter, personID);
 
-		for (Entry<String, Map<String, ?>> entry : this.referenceDatabasesToCreate.entrySet())
+		for (Map<String, ?> refDBMapping : this.referenceDatabasesToCreate.values())
```

Comment:
It's not a new thing, but maybe a comment on what the ? object type could be? Or a general overview of what the object could be?